README.md

```diff
@@ -139,7 +139,11 @@ be specified using the predicate `#delimiter!`.
 )
 ```
 
-Note that this would be the same as `@append_space` with space as delimiter.
+If there is already a semicolon, the `@do_nothing` instruction will be activated
```

Comment:
I thought that the example in the `@do_nothng` section was more or less the same, so I did not feel that it was required to expand it.

Reply:
Ok.

Reply:
Well, in fact, it is exactly the same example, but since it is just under the sentence "If any of the captures in a query match are `@do_nothing`, then the match will be ignored." it does not look weird.